### ISSUE TITLE
BUILD ON OSX "error: no member named 'reference_dts' in 'AVStream'"

### ISSUE THREAD

**Issue by phobax** [bytedeco/javacv#37]
- executed cppbuild.sh and mvn install on javacpp.presets
- executed mvn install on javacpp

mvn install on javacs fails with 

javacv/target/classes/com/googlecode/javacv/cpp/jniavformat.cpp:9833:10: error: no member named 'reference_dts' in 'AVStream'
    ptr->reference_dts = arg0;
    ~~~  ^
javacv/target/classes/com/googlecode/javacv/cpp/jniavformat.cpp:9845:25: error: no member named 'reference_dts' in 'AVStream'
    jlong rvalue = ptr->reference_dts;

ffmpeg version is 2.2
libopencv version is 2.4.8

**Comment by saudet** [comment]
Given the lack of feedback, I'm assuming this issue has since been resolved, but please let me know if this is not the case, thank you!

**Comment by generative-gestaltung** [comment]
hi, problem was not resolved. i didn't have time for further investigations.
thx

**Comment by saudet** [comment]
Well, it is now an old version of FFmpeg. It should be fixed with 2.7... In any case, please repost again if you find that it stills happens with 2.7, thanks!
